web/src/main/resources/org/openmrs/web/filter/update/maintenance.vm

```diff
@@ -1,100 +1,90 @@
 #parse($HEADER_TEMPLATE)
 
-<div class="content-container">
-    <div class="form-container">
-        <form method="post" autocomplete="off" class="maintenance-form">
-            <h2 class="form-title">$l10n.get("update.maintenance.login").insert($superuserrole)</h2>
-
-            <div class="form-field">
-                <label for="username">$l10n.get("update.maintenance.username")</label>
-                <input type="text" id="username" name="username" class="input-field"/>
-            </div>
-
-            <div class="form-field">
-                <label for="password">$l10n.get("install.simple.passwd")</label>
-                <input
-                    type="password"
-                    id="password"
-                    name="password"
-                    class="input-field"
-                    autocomplete="new-password"/>
-            </div>
-
-            <div class="form-actions">
-                <button type="submit" class="button button-primary">
-                    <img src="images/forward_button.png" alt="Continue" class="button-icon"/>
-                </button>
-            </div>
-
-            <input type="hidden" name="page" value="maintenance.vm"/>
-        </form>
-    </div>
+<div class="page-container">
+  <div class="login-box">
+    <h2 class="login-title">System Developer Login</h2>
+    <p class="login-subtext">Log in with a user account that has the <strong>System Developer</strong> role.</p>
```

Comment:
Did you intentionally hard code all this text?

Reply:
will fix this